README.md

```diff
@@ -44,6 +46,24 @@
 
 The example app is a catalogue of the core components provided by this library. With little effort, you can see the functionality of each component using your real data. You can also modify the example code to see how you can customize Discovery Components to fit your needs.
 
+### Setup script
+
+The `runExampleApp.sh` script (located in `examples/discovery-search-app/scripts`) provides prompts to help configure and run the example application. The script iterates through the following steps:
+
+1. Verify all prerequite programs are installed
+1. Prompt user for necessary cluster information
+1. Configure the example application server
+1. Build the React components
+1. Ask the user if they'd like to start the example application
+
+If a user chooses not to start the example application, all previous configuration steps will be left intact so the application can be run at another time by running
```

Comment:
To stay consistent with the rest of the README, I think we should use `you` over `user` here and in the above updates. For example, this sentence would then be something like: `If you choose not to start the example application, all previous...`